contents/blog/why-product-engineering-is-so-fun.md

```diff
@@ -0,0 +1,98 @@
+---
+date: 2022-02-06
+title: Why I've had more fun as a product engineer than any other role in tech
```

Comment:
```suggestion
title: I've had more fun as a product engineer than any other role in tech
```

I really like the tone of this headline, but I'm trying to think of a way to make it easier to parse. Maybe it doesn't matter? Interested in your input here as well @ivanagas + @charlescook-ph.

Reply:
I know "Why" posts can be very click-y but I'm fine without it, you all make the call 👍

Reply:
The intention is fine, I think it is just too long. Should probably be some combination of "product engineer", "fun/best/favorite", and "tech role". Some ideas: 
- Why being a product engineer is the best role in tech
- Why product engineer is my favorite tech role
- I'm a product engineer, here's why it's the best
- Product engineer is the most fun role in tech